dead_mans_switch_client/models/dead_mans_switch_client.py

```diff
@@ -66,3 +67,19 @@ def alive(self):
                 {
                     'Content-Type': 'application/json',
                 }))
+
+    @api.model
+    def _install_default_url(self):
+        """Set up a default URL."""
+        conf = self.env["ir.config_parameter"]
+        name = "dead_mans_switch_client.url"
+        param = conf.get_param(name)
+
+        if not param:
+            url = "{}/dead_mans_switch/alive".format(
+                conf.get_param(
+                    "report.url",
+                    conf.get_param(
+                        "web.base.url",
+                        "http://localhost")))
```

Comment:
It's what I'm doing here. The `http://localhost` part is only for when `web.base.url` is not available.